### ISSUE TITLE
DSEC-split

### ISSUE THREAD

**Issue by ZhouQianang** [ispc-lab/TMA#3]
Hello, thank you for your amazing work!
I have a question, will there be a significant difference between the results obtained in `DSEC-split` mode compared to training the whole set and submitting? I would like to submit cautiously to minimize the number of submissions.

**Comment by HTLeoo** [comment]
Hi,
The split should only be used for model improvements.
The reported EPE is 0.74. But on the split, the EPE is about 0.59. The gap is big.

**Comment by ZhouQianang** [comment]
> Hi, The split should only be used for model improvements. The reported EPE is 0.74. But on the split, the EPE is about 0.59. The gap is big.

Thank you for your reply. To clarify, can I understand it as that: 
Under the same training strategy (`learning rate=2e-4`, `batch_size=6`, `total steps=20000`), training on `dsec-split-train` and testing on `dsec-split-test`, the EPE is about 0.59.

**Comment by HTLeoo** [comment]
![image](https://github.com/ispc-lab/TMA/assets/53611983/708f94a2-833c-4c77-ae44-e2fcc253f9f4)
Yes, you are right.

**Comment by ZhouQianang** [comment]
> ![image](https://private-user-images.githubusercontent.com/53611983/293120618-708f94a2-833c-4c77-ae44-e2fcc253f9f4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM3MjgxMTAsIm5iZiI6MTcwMzcyNzgxMCwicGF0aCI6Ii81MzYxMTk4My8yOTMxMjA2MTgtNzA4Zjk0YTItODMzYy00Yzc3LWFlNDQtZTJmY2MyNTNmOWY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjI4VDAxNDMzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRlMmQwY2E4NTgzNTIyNDQyYWViNTk5Mjc3YjE2ZjRkZTQ4NTYzMmQ3NDAyMGQ5ZDMyODhlNDE0MmRkNGI5ZWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1IZFYIMlyv94Dyndht8QF9IEh1IPUHuIGs8Eb7fig5Q) Yes, you are right.

Thank you again! orz

**Comment by ZhouQianang** [comment]
> Hi, The split should only be used for model improvements. The reported EPE is 0.74. But on the split, the EPE is about 0.59. The gap is big.

Hello! May I have your email, my email is `zqa1313@mail.ustc.edu.cn`, we can talk about, thank you!
